### DISCUSSION TITLE
Help with TTS config

### DISCUSSION THREAD

**Discussion by unstuckconsortium** [OpenVoiceOS/ovos-buildroot#200] [Q&A] [answered]
Hello.  I've been working to build an AI Assistant for a couple of years.  I started out with Mycroft, which I liked, but I really wanted my own WakeWord (ww) and it was near impossible for me to do as it stood pre Mimic 3.  After setting it down for several months, I stumbled upon OVOS,. .and I'm excited to see there is an active community.
Once I have the full build, I plan to document what I've done, and maybe make a YouTube (YT) video (or 2) showing the full process, which includes the hardware installation etc.  So very happy to share once I've got everything figured out.
I have managed to take the Precise WW that I created and convert it to PreciseLite, which amazingly works.  Where I'm stuck now is the TTS.  I really want the voice to be female.  Preferably GB, but US is fine.  I am trying to stay in the OVOS suites, using ovos-tts-server and ovos-tts-plugin-server (and it then has ovos-tts-plugin-mimic and this has me confused).  
When I go to start up the tts-server, I get errors or non responding output that I'd expect.  And I get a lot of "Mycroft is depreciated" messages.
Another thing to mention here,.. that I'm keen to manage the TTS either on my own local server, or on the system itself.  With the least amount of Internet interactions as possible.  I have several reasons for this, but it's what I need for the build I'm making.
Has anyone with a working system got suggestions for me that would help me setup the TTS, including the mycroft.conf config?
I'd appreciate any help.  I really think that once I've got the TTS setup the way I want, I'll be most of the way to where I need this to be.
Appreciate your help and support.
![Screenshot_20231206_234722](https://github.com/OpenVoiceOS/ovos-buildroot/assets/125754887/75ec0c05-eb71-4698-93e8-facf87226aa5)

Thanks,
Bryan

**Comment by goldyfruit** [comment, marked as answer]
Hello,

The Piper or Mimic3 TTS public servers don't have GB female voices.

- Piper voices: https://github.com/OpenVoiceOS/ovos-tts-plugin-piper/blob/dev/ovos_tts_plugin_piper/__init__.py#L38-L198
- Mimic3 voices: https://github.com/OpenVoiceOS/ovos-tts-plugin-mimic3/blob/dev/ovos_tts_plugin_mimic3/__init__.py#L257-L941

With Piper the `en-us` female voice is `amy-low` but I really like the male one `ryan-high`.

Here will be a `mycroft.conf` configuration example based on your initial post.

```json
{
  "log_level": "INFO",
  "logs": {
    "path": "stdout"
  },
  "lang": "en-us",
  "listener": {
    "remove_silence": true,
    "VAD": {
      "module": "ovos-vad-plugin-silero"
    }
  },
  "tts": {
    "module": "ovos-tts-plugin-server",
    "ovos-tts-plugin-server": {
      "voice": "amy-low"
    }
  }
}
```

By default the `ovos-tts-plugin-server` plugin will reach the public Piper TTS servers (https://github.com/OpenVoiceOS/status).

Maybe some other TTS such as Coqui as GB female voices.

Its better to jump on Matrix for "live" support: https://matrix.to/#/#openvoiceos:matrix.org

**Reply by JarbasAl** [reply]
piper seems to have a female gb `southern_english_female-low`

**Reply by unstuckconsortium** [reply]
Thanks to both of you,.. I'll try this out.
